### PR TITLE
Sort out argument parsing

### PR DESCRIPTION
**Pull request by kingdom5500**
Empty containers no longer brick the commands
image
image
It now no longer allows for 'weird' characters :D 
image
Also added fancy random titles for when the bot is harassed.
image
image
I think that pretty much explains this PR

### CODE REVIEW

File: bot/cogs/tags.py [hunk 1]
```diff
@@ -17,6 +18,19 @@ class Tags:
     Save new tags and fetch existing tags.
     """
 
+    FAIL_TITLES = [
+        "Please don't do that.",
+        "You silly tart.",
```
Comment (gdude2002) [inline bot/cogs/tags.py]: Probably remove this specific one.

File: bot/cogs/tags.py [hunk 2]
```diff
@@ -107,43 +121,37 @@ def is_number(string):
...
                         "Rejecting the request.")
-            embed.title = "Please don't do that"
+            embed.title = random.choice(Tags.FAIL_TITLES)
             embed.description = "Tag names can't be numbers."
 
+        # 'tag_name' is longer than 127 characters
+        elif len(tag_name) > 127:
+            log.warning(f"{author} tried to request a tag name with over 127 characters. "
+                        "Rejecting the request.")
+            embed.title = random.choice(Tags.FAIL_TITLES)
```
Comment (gdude2002) [inline bot/cogs/tags.py]: Surely you could set the title once?

File: bot/cogs/tags.py [hunk 3]
```diff
@@ -237,14 +247,14 @@ def _command_on_cooldown(tag_name) -> bool:
 
             if isinstance(tag_data, dict):
                 log.warning(f"{ctx.author} requested the tag '{tag_name}', but it could not be found.")
-                embed.description = f"Unknown tag: **{tag_name}**"
+                embed.description = f"There wasn't a **{tag_name}** tag! Make sure you typed it correctly."
```
Comment (gdude2002) [inline bot/cogs/tags.py]: I prefer the message that was already here.

File: bot/cogs/tags.py [hunk 4]
```diff
@@ -237,14 +247,14 @@ def _command_on_cooldown(tag_name) -> bool:
...
             else:
                 log.warning(f"{ctx.author} requested a list of all tags, but the tags database was empty!")
                 embed.description = "**There are no tags in the database!**"
 
             if tag_name:
                 embed.set_footer(text="To show a list of all tags, use bot.tags.get().")
-                embed.title = "Tag not found"
+                embed.title = "Couldn't find that tag!"
```
Comment (gdude2002) [inline bot/cogs/tags.py]: I prefer the message that was already here.

File: tox.ini [hunk 5]
```diff
@@ -2,5 +2,5 @@
 max-line-length=120
 application_import_names=bot
 exclude=.venv
-ignore=B311,W503,E226
+ignore=B311,W503,E226,S311
```
Comment (gdude2002) [inline tox.ini]: What error is this? Why are we ignoring it?

File: bot/cogs/tags.py (hunk 3 above)
Comment (kingdom5500) [reply bot/cogs/tags.py]: image
lemon doesn't.

File: tox.ini (hunk 5 above)
Comment (kingdom5500) [reply tox.ini]: it stops travis from whining about `random.choice` being insecure

File: bot/cogs/tags.py (hunk 3 above)
Comment (gdude2002) [reply bot/cogs/tags.py]: This was my review, not lemon's. He's free to comment as well.

File: tox.ini (hunk 5 above)
Comment (gdude2002) [reply tox.ini]: Ah, okay

File: bot/cogs/tags.py (hunk 1 above)
Comment (kingdom5500) [reply bot/cogs/tags.py]: oh damn it, you silly tart >.>
<.<
jk c:

**Comment by gdude2002** [comment]
Ah, okay, I see your point.

File: bot/cogs/tags.py (hunk 3 above)
Comment (kingdom5500) [reply bot/cogs/tags.py]: of course ^^

**Comment by 1mn** [comment]
looks good to me, we can probably merge this.

**Pull request merged**